adala/runtimes/_litellm.py

```diff
@@ -158,12 +159,31 @@ def record_to_record(
                 # extra inference params passed to this runtime
                 **self.model_extra,
             )
+        except InstructorRetryException as e:
+            # get root cause error from retries
+            n_attempts = e.n_attempts
+            e = e.__cause__.last_attempt.exception()
+            logger.error(f"Inference error {e} after {n_attempts=}")
+            error_message = type(e).__name__
+            error_details = str(e)
+            tb = traceback.format_exc()
+            if self.verbose:
```

Comment:
I know I fought for keeping this in a while ago, but if we're doing a logger.error anyway, I figure this is somewhat redundant at that point? Feels more natural to use logging levels, and just print the traceback at the debug level, I think. If we care about rich (iirc that's all print_error is adding to print), then we can use the rick logging handler.

Reply:
yep, getting rid of the verbose setting here

maybe later it'd make sense to just have that set the log level...